tools/projmgr/docs/Overview.md

```diff
@@ -0,0 +1,743 @@
+# [DRAFT] Overview
```

Comment:
Content seems more or less duplicated with the one from docs/Manual/Overview.md

Reply:
correct, we are in the process to organize it.